apps/vscode-extension/package.json

```diff
@@ -18,6 +18,15 @@
     "configuration": {
       "title": "Otter",
       "properties": {
+        "otter.packageManager": {
+          "type": "string",
+          "default": "npm",
```

Comment:
To be alignes with the default value of vscode and the default values of big part of vscode extension.
I was thinking it would make more sense in the extension to follow the standard over our specific case for default values (because they are easy to override in the repo 😉)